types/lodash/index.d.ts

```diff
@@ -7221,296 +7257,879 @@ declare namespace _ {
+    interface LoDashExplicitWrapperBase<T, TWrapper> {
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: ListIterator<T, boolean>,
+            fromIndex?: number
+        ): any;
+
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: string,
+            fromIndex?: number
+        ): any;
+
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: PartialObject<T>,
+            fromIndex?: number
+        ): any;
+    }
+
+    //_.flatMap
+    interface LoDashStatic {
+        /**
+         * Creates an array of flattened values by running each element in collection through iteratee
+         * and concating its result to the other mapped values. The iteratee is invoked with three arguments:
+         * (value, index|key, collection).
+         *
+         * @param collection The collection to iterate over.
+         * @param iteratee The function invoked per iteration.
+         * @return Returns the new flattened array.
+         */
+        flatMap<T, TResult>(
+            collection: List<T> | null | undefined,
+            iteratee?: ListIterator<T, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TResult>(
+            collection: List<any> | null | undefined,
+            iteratee?: ListIterator<any, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<T, TResult>(
+            collection: Dictionary<T> | null | undefined,
+            iteratee?: DictionaryIterator<T, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TResult>(
+            collection: Dictionary<any> | null | undefined,
+            iteratee?: DictionaryIterator<any, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<T, TResult>(
+            collection: NumericDictionary<T> | null | undefined,
+            iteratee?: NumericDictionaryIterator<T, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TResult>(
+            collection: NumericDictionary<any> | null | undefined,
+            iteratee?: NumericDictionaryIterator<any, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TObject extends Object, TResult>(
+            collection: TObject | null | undefined,
+            iteratee?: ObjectIterator<any, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TResult>(
+            collection: Object | null | undefined,
+            iteratee?: ObjectIterator<any, Many<TResult>>
+        ): TResult[];
+
+        /**
+         * @see _.flatMap
+         */
+        flatMap<TWhere extends Object, TObject extends Object>(
```

Comment:
See above comment -- this overload basically takes `any` twice, making any future overloads unreachable (without explicit type arguments).